### ISSUE TITLE
how to resume training Process.

### ISSUE THREAD

**Issue by youzarsifbutt** [affinelayer/pix2pix-tensorflow#133]
i have train pix-2-pix algorithms for 200 epoch. i want to train it for 100 more epochs. can you tell me how to resume training process.   @christopherhesse  @leberknecht

**Comment by davesean** [comment]
> --checkpoint folder_train

When you want to continue training add the "--checkpoint path/to/output_dir" with the corresponding folder that was the output_dir the first time you ran it.

**Comment by dustyYMelody7** [comment]
> > --checkpoint folder_train
> 
> When you want to continue training add the "--checkpoint path/to/output_dir" with the corresponding folder that was the output_dir the first time you ran it.

Do you mean I can continue training the network just by adding the checkpoint parameter in following code:???
 python pix2pix.py \
  --mode train \
  --output_dir facades_train \
  --max_epochs 200 \
  --input_dir facades/train \
  --which_direction BtoA
【add here：--checkpoint ....】

**Comment by davesean** [comment]
Yes, that is correct. The folder that you specify needs to include the checkpoint that was previously saved.
